Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -1232,12 +1232,17 @@ auto WebProcessProxy::visiblePageToken() const -> VisibleWebPageToken
 void WebProcessProxy::addPreviouslyApprovedFileURL(const URL& url)
 {
     ASSERT(url.isLocalFile());
-    m_previouslyApprovedFilePaths.add(url.fileSystemPath());
+    auto fileSystemPath = url.fileSystemPath();
+    if (!fileSystemPath.isEmpty())
+        m_previouslyApprovedFilePaths.add(fileSystemPath);
 }
 
 bool WebProcessProxy::wasPreviouslyApprovedFileURL(const URL& url) const
 {
     ASSERT(url.isLocalFile());
+    auto fileSystemPath = url.fileSystemPath();
+    if (fileSystemPath.isEmpty())
+        return false;
     return m_previouslyApprovedFilePaths.contains(url.fileSystemPath());
```

Comment:
Should reuse the `fileSystemPath` variable.

Reply:
whoops, missed that, fixing